fake-server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const bodyParser = require("body-parser");
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post('/', (req, res) => {
+    console.log(req.body);
+    let { username, password } = req.body || {};

```

Comment:
Why is `let` being used here? `username` and `password` are not reassigned so should be using `const` instead.

Reply:
good idea

Reply:
this would also have prevented the assignment to password bug I had introduced below!